modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/v1_x/FeatureTypeInfoImpl.java

```diff
@@ -77,7 +77,7 @@ public String getDescription() {
 
     @Override
     public String getName() {
-        return eType.getName().getLocalPart();
+        return eType.getName().getPrefix() + "_" + eType.getName().getLocalPart();
```

Comment:
This seems odd to me as the prefix is used to line up with a namespace and not strictly speaking part of the name.

Looking at the API this information is available:

* ResourceInfo.getSchame(): provides the namespace
* ResourceInfo.getName() provides the Name

Do we have a solid reason to include the prefix here? This is "just" for the info object and should match up with what DataStore.getFeatureSource( String ) and getFeatureSource( Name ) provides.

In this case, since WFS is "namespace aware" I would expect a user to construct a new NameImpl( res.getSchema(), res.getName() ) to look up a FeatureSource.